client/src/components/AlumniContactModal.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react';
+import {Button, Modal, Image, Grid, Form } from 'semantic-ui-react';
+import swal from "sweetalert";
+import { makeCall } from "../apis";
+
+/*
+DETAILS: Modal for starting/ updating a conversation between alumni
+props:
+    - userDetails: logged in user profile
+    - role: user role
+    - modalOpen: boolean
+    - closeModal: ()
+    - alumni (contains all data from alumni schema)
+    - id
+*/
+export default class AlumniContactModal extends Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            alumni: null,
+            note: '',
```

Comment:
nit: can we have this be `message`, since we use `note` on the frontend to mean something different?